typescript/segmented-cfmm/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "segmented-cfmm-sdk",
+  "version": "1.0.0",
+  "description": "A Typescript SDK to interact with Segmtend CFMM contracts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "api-extract-init": "api-extractor init",
+    "api-extract": "api-extractor run --local --verbose"
+  },
+  "devDependencies": {
+    "typescript": "^4.1.5"
+  },
+  "dependencies": {
+    "@microsoft/api-extractor": "^7.13.1",
+    "@taquito/michel-codec": "^8.0.6-beta.0",
+    "@taquito/signer": "^8.0.4-beta-RC.0",
+    "@taquito/taquito": "^8.0.4-beta-RC.0"
```

Comment:
I think these dependencies are a bit outdated, we should use newer versions if possible

Reply:
I've updated to `10.0.0` which is the latest version of taquito.